docs/touchablewithoutfeedback.md

```diff
@@ -165,9 +165,9 @@ Describes the current state of a component to the user of an assistive technolog
 
 See the [Accessibility guide](accessibility.md#accessibilitystate-ios-android) for more information.
 
-| Type                                                                                           | Required |
-| ---------------------------------------------------------------------------------------------- | -------- |
-| object: {disabled: bool, selected: bool, checked: bool or 'mixed', busy: bool, expanded: bool} | No       |
+| Type                                                                                           |
+| ---------------------------------------------------------------------------------------------- |
+| object: {disabled: bool, selected: bool, checked: bool or 'mixed', busy: bool, expanded: bool} |
```

Comment:
In the future, in separate PR it would be nice to extract this object definition to the separate page, just like for example `ViewToken` right now (https://reactnative.dev/docs/next/viewtoken), because it's is used in multiple components (Touchables, Button etc.):
* https://github.com/facebook/react-native/blob/main/Libraries/Components/View/ViewAccessibility.js#L59:L66